### PR TITLE
A few correction and adding missing actors

### PR DESCRIPTION
**Pull request by DuyGamerLHB**
Added the one-legged variant of the stomped zombieman and sergeant
Fixed the dead zombies spawning the wrong actors due to their missing death states

### CODE REVIEW

File: Zombiemen.txt [hunk 1]
```diff
@@ -1647,12 +1657,37 @@ ACTOR VeryFuckedUpBrutalizedZombiemanLeg2: VeryFuckedUpBrutalizedZombiemanLeg
...
 		Goto Spawn
-		Spawn:
+	Spawn:
         01CB F -1
-        Stop}}
+        Stop
+	}
+}
+
+ACTOR StompedZombieman_NoLeg: DeadZombieman_NoHead
```
Comment (BLOODWOLF333) [inline Zombiemen.txt:1679]: StompedZombieman and StompedZombieman_NoLeg are the same thing, theyre both using frame 01CBF so why did you need to make this duplicate _NoLeg actor?  Its redundant.

File: Zombiemen.txt [hunk 2]
```diff
@@ -1558,16 +1558,8 @@ Actor DeadZombieMan_NoArm: ZombiemanTorso
 	Spawn:
         POS3 E -1
         Stop
-
```
Comment (BLOODWOLF333) [inline Zombiemen.txt:1561]: What is the point of this?  Why?

File: Zombiemen.txt [hunk 3]
```diff
@@ -1580,7 +1572,25 @@ Actor DeadZombieMan_NoLeg: DeadZombieMan_NoArm
 			TNT1 A 0 A_SpawnItem("MuchBlood", 0, 10)
 			TNT1 A 0 A_CustomMissile ("XDeathZombieLeg", 15, 0, random (0, 360), 2, random (0, 180))
 			Stop
-    }}
+    }}
+	
+Actor DeadZombieMan_NoLegNoHead: DeadZombieMan_NoArm
```
Comment (BLOODWOLF333) [inline Zombiemen.txt:1586]: Same things as DeadZombieMan_NoLeg using the same POS7D frame, this is a redundant actor

**Comment by DuyGamerLHB** [comment]
> Dont create redundant actors I dont want that

I removed them now, and the dead zombieman with no leg and no head now has its own sprite

**Review by BLOODWOLF333 — APPROVED** [review]

**Review by BLOODWOLF333 — APPROVED** [review]
